### PR TITLE
Fix(Docs): Improve readme readability

### PR DESCRIPTION
**Pull request by mirko314**
- Adds a quickstart guide to the github frontpage
- Unifies the code examples and make it simpler to start with them
- Link to web/node readmes

### CODE REVIEW

File: packages/web/README.md [hunk 1]
```diff
@@ -37,15 +37,23 @@ npm install @imgly/background-removal
 
 ## Usage
 
-```typescript
-import imglyRemoveBackground from "@imgly/background-removal"
```
Comment (DanielHauschildt) [inline packages/web/README.md:41]: Why did you remove the import statement completely? Wouldn't we need it?

File: packages/web/README.md [hunk 2]
```diff
@@ -37,15 +37,23 @@ npm install @imgly/background-removal
...
-
-let image_src: ImageData | ArrayBuffer | Uint8Array | Blob | URL | string = ...;
-
-imglyRemoveBackground(image_src).then((blob: Blob) => {
-  // The result is a blob encoded as PNG. It can be converted to an URL to be used as HTMLImage.src
-  const url = URL.createObjectURL(blob);
-})
+```js
+async function demoRemoveBackground(url) {
+  console.log('Starting background removal');
```
Comment (DanielHauschildt) [inline packages/web/README.md:42]: Do we really want console outputs here

File: packages/web/README.md [hunk 3]
```diff
@@ -37,15 +37,23 @@ npm install @imgly/background-removal
...
+  const imageBlob = await removeBackground(url, {
+    progress: (key, current, total) => {
+      const [type, subtype] = key.split(':');
+      console.log(
```
Comment (DanielHauschildt) [inline packages/web/README.md:46]: I am not sure about console outputs

File: packages/web/README.md [hunk 4]
```diff
@@ -37,15 +37,23 @@ npm install @imgly/background-removal
...
+        `${type} ${subtype} ${((current / total) * 100).toFixed(0)}%`
+      );
+    }
+  });
+  const newUrl = URL.createObjectURL(imageBlob);
+  console.log('Image with background removed accessible at: ' + newUrl);
+}
+demoRemoveBackground(
+  'https://images.unsplash.com/photo-1686002359940-6a51b0d64f68?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1024&q=80'
```
Comment (DanielHauschildt) [inline packages/web/README.md:55]: Should we really have unsplash links in here. I would rather remove that

File: packages/node/README.md [hunk 5]
```diff
@@ -34,15 +34,27 @@ npm install @imgly/background-removal-node
...
 ## Usage
 
-```typescript
-import {removeBackground} from "@imgly/background-removal-node";
-// const {removeBackground} = require("@imgly/background-removal-node");
-
-let image_src: ImageData | ArrayBuffer | Uint8Array | Blob | URL | string = ...;
+```js
+const { removeBackground } = require('@imgly/background-removal-node');
+const fs = require('fs');
```
Comment (DanielHauschildt) [inline packages/node/README.md:39]: there is 'fs/promises'. I think that's better then fs.promises usage later

File: packages/node/README.md [hunk 6]
```diff
@@ -34,15 +34,27 @@ npm install @imgly/background-removal-node
...
+
+async function demoRemoveBackground(imageUrl) {
+  console.log('Removing background from: ' + imageUrl);
+  const blob = await removeBackground(imageUrl, { debug: false });
+  const buffer = await blob.arrayBuffer();
+  try {
+    await fs.promises.mkdir('tmp', { recursive: true });
+    await fs.promises.writeFile('tmp/output.png', Buffer.from(buffer));
+    console.log('Image saved to tmp/output.png');
```
Comment (DanielHauschildt) [inline packages/node/README.md:48]: I am not sure about console outputs here

File: packages/node/README.md [hunk 7]
```diff
@@ -34,15 +34,27 @@ npm install @imgly/background-removal-node
...
+  } catch (error) {
+    console.error(error);
+  }
+}
 
-removeBackground(image_src).then((blob: Blob) => {
-  // The result is a blob encoded as PNG. It can be converted to an URL to be used as HTMLImage.src
-})
+demoRemoveBackground(
+  'https://images.unsplash.com/photo-1686002359940-6a51b0d64f68?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1024&q=80'
```
Comment (DanielHauschildt) [inline packages/node/README.md:55]: Unsplash links in demo code? Is that common?

File: README.md [hunk 8]
```diff
@@ -16,6 +16,66 @@ Explore an [interactive demo](https://img.ly/showcases/cesdk/web/background-remo
...
+
+Note: On the first run the wasm and onnx model files are fetched. This might, depending on the bandwidth, take time. Therefore, the first run takes proportionally longer than each consecutive run. Also, all files are cached by the browser and an additional model cache.
+
+### Browser
+
+- Run `npm i @imgly/background-removal` to install the package.
+- Run the following code:
+
+  ```js
+  async function demoRemoveBackground(url) {
```
Comment (DanielHauschildt) [inline README.md:29]: See comments in the indivudal files. Import statements seems missin

**Review by DanielHauschildt — CHANGES_REQUESTED** [review]

File: packages/web/README.md (hunk 4 above)
Comment (mirko314) [reply packages/web/README.md:55]: Would you rather host example images yourself somewhere and link them here? Since we are not sure which framework/build system the user is using, we can not use local images.

Comment (mirko314) [reply packages/web/README.md:55]: And I would love if the quick start example code "just works"

File: packages/node/README.md (hunk 6 above)
Comment (mirko314) [reply packages/node/README.md:48]: Personally I find them helpful when nothing is visibly happening.

File: packages/node/README.md (hunk 5 above)
Comment (mirko314) [reply packages/node/README.md]: Changed
